apps/src/componentLibrary/checkboxDropdown/CheckboxDropdown.tsx

```diff
@@ -26,6 +26,12 @@ import {
 } from '@cdo/apps/componentLibrary/common/contexts/DropdownContext';
 import {dropdownColors} from '@cdo/apps/componentLibrary/common/constants';
 
+export interface CheckboxOption {
```

Comment:
This wasn't exported for use, so I added it because I have a lot of functions creating filters in TypeScript that benefitted from it. @levadadenys Is it okay that I added it?